packages/devtools/.env.development

```diff
@@ -1,7 +1,6 @@
+GANACHE_CACHE_FOLDER = '../../.ganache-deployments'
```

Comment:
Can no longer use `source` to load `.env` files when there are spaces around `= `

Reply:
Do we ever use `source`?

Reply:
Good question. We have in the past. Not sure we do at this point? If others do not use source, I am ok losing that functionality if this PR gives us a benefit like syntax highlighting.

Reply:
There is also an argument for readability (regardless of syntax highlighting) with this PR.